client/components/main/index.jsx

```diff
@@ -11,7 +11,11 @@ module.exports = React.createClass( {
 
 	render: function() {
 		return (
-			<main className={ classnames( this.props.className, 'main' ) } role="main">
+			<main
+				className={ classnames( this.props.className, 'main' ) }
+				role="main"
+				style={ this.props.style }
+			>
 				{ this.props.children }
 			</main>
```

Comment:
This is somewhat of a personal preference, but I like keeping `this.props` out of the return area in `render` Not worth blocking a PR for, but keeps the view clean

```js
render() {
	const {
		children
		className,
		style
	} = this.props;

	return (
		<main { ...{ className, style } }>
			{ children }
		</main>
	);
}
```